lib/include/public/mat.h

```diff
@@ -576,6 +577,22 @@ extern "C" {
         ctx.handle = handle;
         return evt_api_call(&ctx);
     }
+
+    /** <summary>
+     * Provide a hint to telemetry system to attempt force-upload of events
+     * based on provided tear down interval before shutdown. This API does not
+     * guarantee the upload.
+     * </summary>
```

Comment:
Change the summary to:
`Flush any pending telemetry events in memory to disk, attempt upload of events if tear down interval is configured, and eventually tear down the telemetry logging system.`